### PR TITLE
Refactor unit test for 'ViewTransfersController'

### PR DESCRIPTION
**Pull request by TomJKing**
The refactor is designed to:
- keep all the expected outcomes in a single table instead of being spread over multiple different vals in the code;
- make it easier to see what status generates what transfer action;
- make it easier see which particular status and action are failing, by individually testing eash of the combinations, instead of testing all together;
- make adding new statuses easier to test

### CODE REVIEW

File: test/testUtils/ConsignmentStatusOptions.scala [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    ("upload in progress", CurrentStatus(None, None, inProgress, None, None, None), "/upload", "In Progress", "Resume transfer"),
+    ("upload failed", CurrentStatus(None, None, failed, None, None, None), "/upload", "Failed", "View errors"),
+    ("upload completed", CurrentStatus(None, None, completed, None, None, None), "/upload", "In Progress", "Resume transfer"),
+    ("client checks in progress", CurrentStatus(None, None, completed, inProgress, None, None), "/file-checks", "In Progress", "Resume transfer"),
+    ("client checks failed", CurrentStatus(None, None, completed, failed, None, None), "/file-checks-results", "Failed", "View errors"),
+    ("client checks completed with issues", CurrentStatus(None, None, completed, completedWithIssues, None, None), "/file-checks-results", "Failed", "View errors"),
+    ("client checks completed", CurrentStatus(None, None, completed, completed, None, None), "/file-checks-results", "In Progress", "Resume transfer"),
+    ("confirm transfer completed", CurrentStatus(None, None, completed, completed, None, None), "/file-checks-results", "In Progress", "Resume transfer"),
```
Comment (thanhz) [inline test/testUtils/ConsignmentStatusOptions.scala:61]: The "confirm transfer completed" state looks the same as the "client checks completed" test. Did you mean to do this instead `CurrentStatus(completed, completed, completed, completed, completed, None)`

**Review by thanhz — COMMENTED** [review]
Thanks, this is much better. :1st_place_medal: Just a few minor things

File: test/controllers/ViewTransfersControllerSpec.scala [hunk 2]
```diff
@@ -30,13 +30,66 @@ class ViewTransfersControllerSpec extends FrontEndTestHelper {
   val checkPageForStaticElements = new CheckPageForStaticElements
 
   private val formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm:ss")
+  private val defaultCurrentStatus = CurrentStatus(None, None, None, None, None, None)
```
Comment (thanhz) [inline test/controllers/ViewTransfersControllerSpec.scala:33]: Naming the arguments here might help with understanding what the statuses are i.e.
`CurrentStatus(series = None, transferAgreement = None, upload = None, clientChecks = None, confirmTransfer = None, export = None)` There are other places with this too but feel free to ignore as they look weird when formatted.

File: test/controllers/ViewTransfersControllerSpec.scala [hunk 3]
```diff
@@ -30,13 +30,66 @@ class ViewTransfersControllerSpec extends FrontEndTestHelper {
...
 
   implicit val ec: ExecutionContext = ExecutionContext.global
 
+  forAll(ConsignmentStatusesOptions.expectedStandardStatesAndStatuses) { (expectedTransferState, currentStatus, actionUrl, transferState, actionText) =>
+    {
+      s"ViewTransfersController for '$standardType' consignments" should {
+        s"render the '$expectedTransferState' action for the given 'consignment status'" in {
+          setConsignmentTypeResponse(wiremockServer, "standard")
+          val consignment: List[Consignments.Edges] = setConsignmentViewTransfersResponse(wiremockServer, standardType, customStatusValues = List(currentStatus))
+
```
Comment (thanhz) [inline test/controllers/ViewTransfersControllerSpec.scala:45]: unnecessary line break here. There's a few more in the other test cases too i.e. line 119, 143, 168

File: test/testUtils/ConsignmentStatusOptions.scala (hunk 1 above)
Comment (TomJKing) [reply test/testUtils/ConsignmentStatusOptions.scala:61]: Good spot

**Review by thanhz — APPROVED** [review]

**Pull request merged**